style.css

```
@@ -163,235 +149,332 @@
   background: #fff;
   border-radius: 8px;
+  margin-right: 10px;
   padding: 5px;
-  font-style: normal;
+  text-align: center;
+  margin-top: 5px;
+  color: #36b37e;
   font-weight: 600;
   font-size: 12px;
-  line-height: 16px;
-  color: #36b37e;
 }
 
-.recent-work {
-  text-align: center;
-  margin-bottom: 20px;
-  font-weight: 700;
-  font-size: 40px;
-}
+/* skill item end */
 
-.hrs {
-  width: 48px;
-  height: 4px;
-  background: #36b37e;
-  border-radius: 24px;
-  margin-left: 45%;
-  margin-top: 10px;
-}
+/* start form part  */
 
-.skill-ulall {
-  list-style: none;
+.form-input-part {
+  margin-left: 10px;
 }
 
-/* form part css style */
-.form-into-text {
-  font-weight: 700;
-  font-size: 40px;
-  text-align: center;
-  color: #172b4d;
-  margin-bottom: 30px;
+.full-form-part {
+  width: 90%;
+  display: grid;
+  grid-template-columns: 3fr 4fr;
 }
 
-.inside-form-full {
-  width: 100%;
+.master-form {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin-top: 40px;
+  background-image: url(images/form-left-part.png), url(images/form-right-part.png);
+  background-repeat: no-repeat;
+  background-size: 10%;
+  background-position: bottom left, bottom right;
+  background-position-y: 200px, 100px;
+  padding: 40px;
+  margin-bottom: 100px;
 }
 
-.contact {
-  width: 80%;
+.form-text {
   display: flex;
   flex-direction: column;
-  margin: 0 auto;
+  justify-content: center;
+  align-items: center;
+  position: relative;
+}
+
+.image-down-form {
+  width: 130px;
+  height: 105px;
+  position: absolute;
+  bottom: 0;
 }
 
-.fullname,
 .emailAdress {
-  box-sizing: border-box;
-  background: #fff;
-  border: 1px solid #d0d9d4;
-  border-radius: 4px;
-  width: 327px;
+  width: 675px;
+  height: 48px;
+}
+
+.input-fullname {
+  display: flex;
+}
+
+.fullname {
   height: 48px;
+  width: 330px;
+  border-radius: 4px;
   margin-bottom: 10px;
+  margin-right: 10px;
+  border: 1px solid gray;
+}
+
+.FullNameMobile {
+  display: none;
 }
 
 .message {
-  box-sizing: border-box;
-  background: #fff;
-  border: 1px solid #60c095;
-  border-radius: 4px;
-  width: 327px;
   height: 114px;
-  margin-bottom: 10px;
+  width: 684px;
+  border-radius: 4px;
+  margin-top: 20px;
+  border: 1px solid gray;
+  margin-bottom: 15px;
 }
 
-.btnsubmit {
-  width: 129px;
-  height: 48px;
+.form-btn-submit {
   background: #36b37f;
   border-radius: 4px;
-  color: white;
   border: none;
-  margin: 0 auto;
-  display: block;
+  color: white;
+  padding: 5px 10px 5px 10px;
 }
 
+/* end form part */
+
 footer {
-  margin-top: 25px;
-  display: flex;
-  justify-content: center;
-  align-items: center;
-  flex-direction: column;
+  text-align: center;
 }
 
-.dayall {
-  background-image: url(images/footer-bg.png);
-  background-repeat: no-repeat;
-  background-position: bottom right;
-  background-position-y: 595px;
-  margin-right: 0;
+.mail-foorter {
+  display: none;
+}
+
+@media only screen and (max-width: 765px) and (min-width: 375px) {
```

- [ ] As requested before, you need to make small changes in your CSS file to implement mobile first approach. This is an example of how it should look like: 
```suggestion
@media only screen and (min-width: 768px) {
```
> ```
> /* Your CSS styles for mobile screen here (no need to define min-width:375px and max-width:768px as we will add the below)*/
> 
> .header {
> }
> 
> .nav {
> }
> 
> @media screen and (min-width: 768px) {
>   /* Your CSS styles for desktop screens here */
> 
>   .header {
>   }
> 
>   .nav {
>   }
> }
> ```